src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,6 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return (a != b && c != d) || (a == b && c == d && a != c);
```

Comment:
use binary operators to simplify. look at this [https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html](checklist) before send the link

Reply:
done